packages/grid/x-data-grid/src/components/toolbar/GridToolbar.tsx

```diff
@@ -34,18 +35,13 @@ const GridToolbar = React.forwardRef<HTMLDivElement, GridToolbarProps>(function
       <GridToolbarColumnsButton />
       <GridToolbarFilterButton />
       <GridToolbarDensitySelector />
-      <GridToolbarExport csvOptions={csvOptions} printOptions={printOptions} />
+      <GridToolbarExport
+        csvOptions={csvOptions}
+        printOptions={printOptions}
+        excelOptions={excelOptions}
```

Comment:
It is not great to have a reference to Excel here in a community component
But to avoid it we would need a do a premium specific `GridToolbar`

Could you add a comment `Stop mentioning Excel here` so that we remember that it is not great ?